edge/pkg/devicetwin/dmiserver/event_type.go

```diff
@@ -0,0 +1,58 @@
+package dmiserver
+
+import (
+	"time"
+)
+
+const (
+	PropertyType string = "propertyType"
+)
+
+// MsgTwin the structure of device twin.
+type MsgTwin struct {
+	Expected        *TwinValue    `json:"expected,omitempty"`
+	Actual          *TwinValue    `json:"actual,omitempty"`
+	Optional        *bool         `json:"optional,omitempty"`
+	Metadata        *TypeMetadata `json:"metadata,omitempty"`
+	ExpectedVersion *TwinVersion  `json:"expected_version,omitempty"`
+	ActualVersion   *TwinVersion  `json:"actual_version,omitempty"`
```

Comment:
suggest use Camel case for json tag

Reply:
this is defined from the old defination and it will have an impact on user interface if we change it to Camel case.